velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -922,6 +922,19 @@ struct DateTruncFunction : public TimestampWithTimezoneSupport<T> {
     }
   }
 
+  FOLLY_ALWAYS_INLINE Timestamp adjustByArithmetic(
+      Timestamp timestamp,
+      long intervalSeconds,
+      const date::time_zone* timezone) {
+    long seconds = getSeconds(timestamp, timezone);
```

Comment:
It seems seconds could be used as the input of this function, and timestamp and tinmezone could be replaced.